### ISSUE TITLE
Error with the context menu in the bookmarks bar (german language) 

### ISSUE THREAD

**Issue by bastele7** [brave/brave-browser#12844]
When I right-click on a folder in the bookmarks, I get an error in the context menu. This does not happen when I click on a single bookmark (see pictures) or if i change the UI-language to english or french.

The problem exists since I upgraded to the new Brave 1.17.73.

Screenshot: Bookmark folder
https://aws1.discourse-cdn.com/brave/original/3X/2/7/27914a04473a605103895eecb278411f266977dd.png

Screenshot: Single bookmark
https://aws1.discourse-cdn.com/brave/original/3X/2/d/2dbdeaa06bcb2ecab69b8ee4ccca475efc941e61.png

**Comment by rebron** [comment]
cc: @mkarolin @simonhong  Looks like in the first screen shot for DE we're missing `Open all in Private Window` and all the `Open` strings aren't locallized.

**Comment by mkarolin** [comment]
@rebron fixes the problem with the missing context menu item. 

The first two `Open` strings in the context menu have not been localized yet.

**Comment by bastele7** [comment]
@rebron: Thanks for fixing this issue. Now the translation is missing at this menue.
https://github.com/brave/brave-browser/issues/12904

**Comment by LaurenWags** [comment]
Verification **PASSED** on `macOS x64 10.15.7` using the following build:

```
Brave | 1.17.75 Chromium: 87.0.4280.88 (Official Build) (x86_64)
-- | --
Revision | 89e2380a3e36c3464b5dd1302349b1382549290d-refs/branch-heads/4280@{#1761}
OS | macOS Version 10.15.7 (Build 19H15)
```

Reproduced the original issue described via https://github.com/brave/brave-browser/issues/12844 using 1.17.73:


**`Single Bookmark`** | **`Bookmark Folder`**
------------------|------------------
<img width="1216" alt="1 17 73 single bookmark" src="https://user-images.githubusercontent.com/28145373/101039677-e8d3a000-3549-11eb-9d4e-c2ea590954dc.png">  | <img width="1216" alt="1 17 73 multiple bookmarks" src="https://user-images.githubusercontent.com/28145373/101039744-ecffbd80-3549-11eb-9c0e-6cc8ef8caeb5.png">

Confirmed translation is fixed on 1.17.75:

**`Single Bookmark`** | **`Bookmark Folder`**
------------------|------------------
<img width="1215" alt="1 17 75 single bookmark" src="https://user-images.githubusercontent.com/28145373/101040178-0ef94000-354a-11eb-8e79-f05205fc390f.png">  | <img width="1213" alt="1 17 75 multiple bookmarks" src="https://user-images.githubusercontent.com/28145373/101040241-13255d80-354a-11eb-8769-e12ad416dd1e.png">

Reproduced issue from https://github.com/brave/brave-browser/issues/12904 as per screenshots above.

-------------
